en/developer_guide/form_type.rst

```diff
@@ -0,0 +1,311 @@
+Form Type List :
+================
+
+Open Orchestra consists of many modules each comprising a certain number of form, here are the
+list of different form type.
+
+- `cms bundle`_
+- `backofficebundle bundle`_
+- `group bundle`_
+- `user admin bundle`_
+- `display bundle`_
+- `media admin bundle`_
+- `model bundle`_
+- `model interface bundle`_
+- `user bundle`_
+- `workflow function bundle`_
+
+.. _cms bundle:
+
+CMS Bundle :
+------------
+
+.. _backofficebundle bundle:
+
+BackofficeBundle:
+~~~~~~~~~~~~~~~~~
+.. code-block:: yaml
+
+    # open-orchestra-cms-bundle/BackofficeBundle/Form/Type
+
++-----------------------------------+-----------------------------------+-----------------------+
+| Class Name                        | alias / form type name            | Use to                |
++===================================+===================================+=======================+
+| ``AbstractGroupChoiceType``       | ``oo_group_choice``               |                       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``ApiClientType``                 | ``oo_api_client``                 |API Client Form        |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``AreaType``                      | ``oo_area``                       |Area and sub area      |
+|                                   |                                   |template               |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``BlockChoiceType``               | ``oo_block_choice``               |generate au choice     |
+|                                   |                                   |field to the list of   |
+|                                   |                                   |existing block         |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``BlockType``                     | ``oo_block``                      |Block Form             |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``ColorPickerType``               | ``oo_color_picker``               |Add a css class        |
+|                                   |                                   |colopiker on           |
+|                                   |                                   |video type field       |
```

Comment:
You should not mention the video type field here as it can be used anywhere.
Plus more than a css class, you should present it as a Color Selector